extensions/cornerstone-dicom-sr/src/utils/addMeasurement.ts

```diff
@@ -92,70 +95,88 @@ function _getRenderableData(GraphicType, GraphicData, imageId) {
       break;
     case SCOORD_TYPES.CIRCLE: {
       // Todo: write cornerstone3D circle logic
-      const center = { x: GraphicData[0], y: GraphicData[1] };
-      const onPerimeter = { x: GraphicData[2], y: GraphicData[3] };
+      throw new Error('Not implemented in cs3d yet');
```

Comment:
We still need this, we had it before. You can just draw an ellipse with equal major and minor axis sizes.